app/src/main/java/com/example/makeitso/model/service/impl/StorageServiceImpl.kt

```diff
@@ -43,29 +40,31 @@ constructor(private val firestore: FirebaseFirestore, private val auth: AccountS
   override val tasks: Flow<List<Task>>
     get() =
       auth.currentUser.flatMapLatest { user ->
-        currentCollection(user.id).snapshots().map { snapshot -> snapshot.toObjects() }
+        firestore.collection(TASK_COLLECTION).whereEqualTo(USER_ID_FIELD, user.id)
+          .snapshots()
+          .map { snapshot -> snapshot.toObjects() }
```

Comment:
Starting in [Firestore 24.5.0](https://firebase.google.com/support/release-notes/android#firestore-ktx_v24-5-0) we can use `dataObjects()` here:

```suggestion
          .dataObjects()
```

Reply:
Done!